docs/linked-resources/add-linked-resource.md

```diff
@@ -21,7 +21,7 @@ links:
 1. Find the `Applications` card
 1. Click `View` in the `Applications` card to see a list of existing `Applications`
 1. Click `DETAILS` on the application you want to link a server
-1. On the upper-right corner of the application details view, click `...`
+1. On the upper-right corner of the application details view, click `SETTINGS`
```

Comment:
@guipontes7 thanks for updating the docs with this change.
Could you update this PR applying the same change in all the references of "click `...`"?

For a full list of files containing the outdated text, see: https://github.com/search?q=repo%3Adevopness%2Fdevopness%20click%20%60...%60&type=code

Reply:
Yes!

PR updated.